flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CalcITCase.scala

```diff
@@ -363,4 +363,28 @@ class CalcITCase extends StreamingTestBase {
       List("1,HI,1111,true,111","2,HELLO,2222,false,222", "3,HELLO WORLD,3333,true,333")
     assertEquals(expected.sorted, sink.getAppendResults.sorted)
   }
+
+  @Test
+  def testMapGet(): Unit = {
```

Comment:
The fix looks good to me, however, I would suggest to add tests to `org.apache.flink.table.planner.expressions.MapTypeTest`. An integeration test is much heavier than unit test.

Reply:
Thx, I will add a test to MapTypeTest.